joomla/joomla-cms
# [4.0] Password field styling

Pull Request for Issue # .

### Summary of Changes

This PR corrects password field styling in login forms and restores view password functionality in site login module.

### Testing Instructions

Apply patch. Check password field in admin login and site logins (com_users and mod_login). Check that view password functionality works.

### Documentation Changes Required
